inputs/select.vue

```diff
@@ -89,37 +94,58 @@
     props: ['name', 'data', 'schema', 'args'],
     data() {
       return {
-        options: [],
+        listOptions: [],
         isDisabled: false
       };
     },
     mounted() {
       if(this.args.list){
         this.fetchListItems().then( listItems => {
-          this.setOptions(listItems);
-        });
-      } else {
-        this.setOptions();
+          this.listOptions = listItems;
+        }); // todo: ".catch"?
       }
     },
     computed: {
-      safeData() {
-        if (_.isString(this.data)) {
-          return _.find(this.options, (option) => option.value === this.data);
-        } else if (_.isObject(this.data) && !_.isEmpty(this.data)) {
-          return _.reduce(this.options, (safeArray, option) => {
-            if (this.data[option.value] === true) {
-              return safeArray.concat([option]);
-            } else {
-              return safeArray;
-            }
-          }, []);
+      keys () {
+        return {
+          label: 'name', // backwards compatibility I guess?
+          value: 'value',
+          ...this.args.keys
+        }
+      },
+      NULL_OPTION () {
+        return {
+          [this.keys.label]: 'None',
+          [this.keys.value]: null,
+        }
+      }, 
+      // combine arg/prop options, fetched list options, and a null option for non-multiple selects
+      options () {
+        const propOptions = (this.args.options || [])
+        let opts = propOptions.concat(this.listOptions)
+        if (opts.length > 0) {
+          if (!this.args.multiple) {
+            opts = [ this.NULL_OPTION, ...opts ]
+          }
```

Comment:
I'm asking in the nymag meeting whether it is ok to only include the null option when the select list is a single select list

Reply:
what did @nelsonpecora say again? I forget.

Reply:
I think he didn't 100% agree that we could remove it, but I believe we can convince him. Can you think of *any* reason why it should show up in this multiple select list:
![pointless-none](https://user-images.githubusercontent.com/1163/37226727-def98884-23a8-11e8-9165-28e5444fb1d1.gif)


Each option can be toggled on and off, toggling a "None" option independent of the other options is nonsensical. What does it mean when no options are selected at all, vs. when the "None" option is selected? What does it mean when "None" is selected in addition to other options? 

That said, we don't really need to fix this because I think Slate doesn't even have any multiple select lists on our site yet (but we will soon, for the authors list)

Reply:
When I try to think of a reasons to have None, I can think of none.... but maybe that's the purpose of it. So that people know you thought about it. Not that you forgot about it.

Reply:
💥